### PR TITLE
Au+Au 19.6 GeV Run19 centrality definition and isobar trigger efficiency calculation

### PR DESCRIPTION
**Pull request by nigmatkulov**
The modifications affect only physics analysis and not touch general codes.

### CODE REVIEW

**Comment by nigmatkulov** [comment]
Dmitri, is there a simple way to exclude StRoot/PWGTools/ from compilation process (CI Build)?

**Comment by plexoos** [comment]
Something like `cons %PWGTools` may work

File: StRoot/StRefMultCorr/BadRun.h [hunk 1]
```diff
@@ -29,20 +47,32 @@ const Int_t badrun_refmult_2018[nBadRun_refmult_2018] =
...
...5, 20078040, 20078043, 20078046, 20078051, 20078054, 20078057, 20078060, 20078063, 20078067, 20079006, 20079009, 20079013, 20079017, 20079020, 20079023, 20079044, 20080006, 20080009, 20080012, 20080016, 20080020, 20081001, 20081004, 20081007, 20081012, 20081015, 20081018, 20081025, 20082002, 20082005, 20082010, 20082013, 20082016, 20082019, 20082024, 20082029, 20082034, 20082038, 20082047, 20082050, 20082053, 20082056, 20082059, 20082063, 20082066, 20083001, 20083004, 20083019, 20083022, 20083025, 20083029, 20083032, 20083074, 20083077, 20083079,20084001, 20084002, 20084005, 20084009, 20084013, 20084016, 20084022, 20085006, 20085009, 20085017, 20086002, 20086005, 20086056, 20086011, 20086015, 20087008, 20087012, 20087021, 20088005, 20088009, 20088012, 20088030, 20088033, 20088037, 20089003, 20089006, 20089009, 20089012, 20089015, 20089018, 20089028, 20090002, 20090005, 20090008, 20090011, 20090014, 20090017, 20090021, 20090031, 20090048, 20091003, 20091006, 20091009, 20091012, 20091016, 20091019, 20091020, 20092005, 20092012, 20092015, 20092018, 20092021, 20092024, 20092027, 20092030, 20092033, 20092038, 20092053, 20092057, 20093001, 20093005, 20093010, 20093016, 20093025, 20093035
```
Comment (marrbnl) [inline StRoot/StRefMultCorr/BadRun.h:72]: the bad run list for 19.6 GeV is consistent with what's shown here: https://drupal.star.bnl.gov/STAR/pwg/common/bes-ii-run-qa/bes-ii-196-gev-col. For example, 20092054 is missing and a few runs from day 093 are listed here but not on the drupal page.

Comment (nigmatkulov) [reply StRoot/StRefMultCorr/BadRun.h:72]: I did not quite get your message. Do you mean that some bad runs are missing? I checked the one you mentioned (20092054) and it is in the list (line 72)? Could you please elaborate more on that?

**Comment by marrbnl** [comment]
There are a few unnecessary files: 
StRoot/PWGTools/Glauber/star_glauber/temp_gccflags.c
StRoot/PWGTools/Glauber/glauberOut/.o

Also, you have a new StRoot directory here: StRoot/PWGTools/Glauber/star_glauber/StRoot. Would this cause any issues with the existing StRoot?

**Comment by nigmatkulov** [comment]
> 
That was my original question to Dmitri about how to exclude PWGTools from the CI Build.
Currently it leads to CI build errors. Let me try to remove PWGTools from the current commit.

**Pull request closed without merging**